operators/operator_sdk/osdk-pkgman-to-bundle.adoc

```diff
@@ -6,6 +6,7 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 Support for the legacy _package manifest format_ for Operators is removed in {product-title} 4.8 and later. If you have an Operator project that was initially created using the package manifest format, you can use the Operator SDK to migrate the project to the bundle format. The bundle format is the preferred packaging format for Operator Lifecycle Manager (OLM) starting in {product-title} 4.6.
+//Does this need an update?
```

Comment:
@adellape, PTAL?

Reply:
Same reasoning as https://github.com/openshift/openshift-docs/pull/36974/files#r720338676. We also only just introducing the migration tooling in 4.8.

Reply:
I'll tag this one, too. Thanks!